typescript-arrays/main.ts

```diff
@@ -0,0 +1,33 @@
+const array: string[] = ['red', 'white', 'blue'];
+const colors: string[] = array;
+
```

Comment:
Just to note that you don't need this `colors` array that is being assigned the value of your `array` variable. Instruction wise you would just need to name the first array variable `colors`

`const colors: string[] = ['red', 'white', 'blue']`